src/components/ListItem.jsx

```diff
@@ -29,12 +29,37 @@ export function ListItem({ listToken, item }) {
 		}
 	}, [listToken, item]);
 
+	const getTextColor = () => {
+		if (item?.currentEstimate <= 7) {
+			return 'red';
+		} else if (item?.currentEstimate > 7 && item?.currentEstimate === 30) {
+			return 'green';
+		} else if (item?.currentEstimate >= 30) {
+			return 'blue';
+		} else if (item?.currentEstimate > 60) {
+			return 'gray';
+		}
+	};
+
+	const getAria = () => {
+		if (item?.currentEstimate <= 7) {
+			return 'soon';
```

Comment:
I think that the logic here needs to be adjusted: 
See adjusted code below
`
if (item?.currentEstimate <= 7) {
			return 'soon';
		} else if (item?.currentEstimate > 7 && item?.currentEstimate <= 30) {
			return 'kind of soon';
		} else if (item?.currentEstimate > 30 && item?.currentEstimate <= 60) {
			return 'not so soon';
		} else if (item?.currentEstimate > 60) {
			return 'inactive';
		}
`

Reply:
Thank you @mide358 made the changes.